release notes/v0.47.0.md

```diff
@@ -79,12 +94,22 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 - [browser#1040](https://github.com/grafana/xk6-browser/pull/1040) fix the `context.clearCookies()` method to clear all cookies from the current browser context.
 - [browser#1031](https://github.com/grafana/xk6-browser/pull/1031) fix the `expires` field while adding cookies using the `context.addCookie()` method.
 - [browser#1039](https://github.com/grafana/xk6-browser/pull/1039) fix Goja conversions while adding and retrieving cookies.
+- [browser#1038](https://github.com/grafana/xk6-browser/pull/1038) Fix read/write data race for edge case with remote browsers.
+- [browser#1035](https://github.com/grafana/xk6-browser/pull/1035) Refactor `int64` timeout to `time.Duration` which ensures no confusion as to whether a timeout is in milliseconds or seconds.
+- [browser#1034](https://github.com/grafana/xk6-browser/pull/1034) Fix `page.reload` & `page.setContent` to use the default navigation timeout over the default timeout.
+- [browser#1033](https://github.com/grafana/xk6-browser/pull/1033) Fix the `page` timeouts so it is actually used after being set.
 
 ## Maintenance and internal improvements
 
 _Format as `<number> <present_verb> <object>. <credit>`_:
 - _`#2770` Refactors parts of the JS module._
 - [#3342](https://github.com/grafana/k6/pull/3342). Updates xk6-grpc to the latest version. This change brings all the latest fixes and improvements to the experimental gRPC module.
+- [browser#1043](https://github.com/grafana/xk6-browser/pull/1043), [browser#1021](https://github.com/grafana/xk6-browser/pull/1021), [browser#1019](https://github.com/grafana/xk6-browser/pull/1019), [browser#1014](https://github.com/grafana/xk6-browser/pull/1014) Fix tests.
+- [browser#1000](https://github.com/grafana/xk6-browser/pull/1000), [browser#1024](https://github.com/grafana/xk6-browser/pull/1024) Refine issue and PR templates.
+- [browser#1003](https://github.com/grafana/xk6-browser/pull/1003) Internal refactor.
+- [browser#997](https://github.com/grafana/xk6-browser/pull/997) Update readme.
+- [browser#962](https://github.com/grafana/xk6-browser/pull/962) CI fixes.
+- [browser#1009](https://github.com/grafana/xk6-browser/pull/1009), [browser#1010](https://github.com/grafana/xk6-browser/pull/1010) Supporting internal changes.
```

Comment:
Following the pattern used in the rest of the entries, should this be `Support` instead of `Supporting`?
```suggestion
- [browser#1009](https://github.com/grafana/xk6-browser/pull/1009), [browser#1010](https://github.com/grafana/xk6-browser/pull/1010) Support internal changes.
```

Reply:
It feels like the meaning changes? 🤔 Can we put it next to 1003, like this?

```
- [browser#1003](https://github.com/grafana/xk6-browser/pull/1003),[browser#1009](https://github.com/grafana/xk6-browser/pull/1009), [browser#1010](https://github.com/grafana/xk6-browser/pull/1010) Improve internals.
```

Reply:
Yep, agreed with both of you.

I'll change it to what @inancgumus suggested but group them as "Internal changes".

Reply:
Resolved in 38c47235577bb44f007ee3d3c9184b7a9aa07ffc